scripts/orchestrator.js

```diff
@@ -70,14 +71,21 @@ const orchestrator = (tabUrl, websiteUrl, userId) => {
 					githubHunkHighlight(hunk_info_response);
 				}
 			}
-			// for showing all tracked repo
+			// for showing all tracked repo in organisation page
 			else if (
-				(urlObj[3] && urlObj[4] == undefined) ||
+				(urlObj[3] && urlObj[4] == undefined && urlEnd !== 'tab=repositories') ||
```

Comment:
Without this extra condition, a user url like https://github.com/avikalpg?tab=repositories was going in to the else if block that handles adding the logo for organization repos.

In my earlier PR changes, this condition was not needed as the else if block for the user repositories condition was being checked first, but since your review asked me to move the condition to the end to not affect current working, this extra check was needed to ensure the url going into the correct block.

[Review for reference](https://github.com/Alokit-Innovations/chrome-extension/pull/81#discussion_r1384784379)